app/models/classification.rb

```diff
@@ -525,7 +525,12 @@ def find_tag
   end
 
   def save_tag
-    self.tag = Tag.find_or_create_by_classification_name(name, region_id, ns, parent_id)
+    tag_name = Classification.name2tag(name, parent_id, ns)
```

Comment:
Small nit but, couldn't this line be moved into the `if` block?

Reply:
sure, it is leftover from my "backporting" from master where `tag_name` used in `else` block too